server/test/handlers/cancelSpecificAppointment.test.js

```diff
@@ -0,0 +1,118 @@
+const workerTimeslot = require('../../models/db/workerTimeslot');
+const appointmentHandler = require('../../models/handlers/appointment');
+const workerTimeslotHandler = require('../../models/handlers/workerTimeslot');
+
+describe('testing cancellation of specific worker appointments', () => {
+    beforeEach(() => {
+        jest.resetModules();
+    });
+
+    test('no parameters passed in for the method', async () => {
+        // Arrange
+        const testIsCancelled = false;
+
+        // Act
+        const isCancelledSuccessfully = await appointmentHandler.cancelSpecificAppointment();
+
+        // Assert
+        expect(isCancelledSuccessfully).toBe(testIsCancelled);
+    });
+
+    test('null values for appointmentId parameter', async () => {
+        // Arrange
+        const appointmentId = null;
+        const testIsCancelled = false;
+
+        // Act
+        const isCancelledSuccessfully = await appointmentHandler.cancelSpecificAppointment(appointmentId);
+
+        // Assert
+        expect(isCancelledSuccessfully).toBe(testIsCancelled);
+    });
+
+    test('empty string for the appointmentId', async () => {
+        // Arrange
+        const appointmentId = '';
+        const testIsCancelled = false;
+
+        // Act
+        const isCancelledSuccessfully = await appointmentHandler.cancelSpecificAppointment(appointmentId);
+
+        // Assert
+        expect(isCancelledSuccessfully).toBe(testIsCancelled);
+    });
+
+    test('invalid true value for the appointmentId', async () => {
+        // Arrange
+        const appointmentId = true;
+
```

Comment:
Hi Peter,
I would recommend removing the blank line on this line, line 61, 74, 95, and 109 for shorter code files.

Reply:
Removed!